pkg/cmd/server/bootstrappolicy/namespace_policy.go

```diff
@@ -67,6 +67,10 @@ func buildNamespaceRolesAndBindings() (map[string][]rbac.Role, map[string][]rbac
 		DefaultOpenShiftSharedResourcesNamespace,
 		newOriginRoleBinding(OpenshiftSharedResourceViewRoleBindingName, OpenshiftSharedResourceViewRoleName, DefaultOpenShiftSharedResourcesNamespace).Groups(AuthenticatedGroup).BindingOrDie())
 
+	addNamespaceRoleBinding(namespaceRoleBindings,
+		DefaultOpenShiftNodeNamespace,
+		newOriginRoleBindingForClusterRole(NodeConfigReaderRoleBindingName, NodeConfigReaderRoleName, DefaultOpenShiftNodeNamespace).Groups(NodesGroup).BindingOrDie())
```

Comment:
Use the RBAC builder so the role binding name is not required (and its clear that its the same as the role name).

Reply:
example location?

Reply:
`rbac.NewRoleBinding(NodeConfigReaderRoleName, DefaultOpenShiftNodeNamespace).Groups(NodesGroup).BindingOrDie())`

Then you can delete `NodeConfigReaderRoleBindingName`

And create the role using `addNamespaceRole`